torch/onnx/symbolic_opset9.py

```diff
@@ -4085,11 +4085,14 @@ def rand_like(
         "RandomUniformLike", self, dtype_i=symbolic_helper.scalar_type_to_onnx[dtype]
     )
 
-
 @symbolic_helper.parse_args("v", "f", "f", "i", "none")
 def rrelu(g, input, lower, upper, training, generator):
-    p = g.op("RandomUniformLike", input, high_f=upper, low_f=lower)
-    return g.op("PRelu", input, p)
+    if not training:
+        slope = (upper + lower) / 2.0
+        return g.op("LeakyRelu", input, alpha_f=slope)
+    else:
```

Comment:
Not need for the `else:` branch after returning

Reply:
Done